components/Collections/examples/CollectionsEditingExample.m

```diff
@@ -87,6 +92,32 @@ - (UICollectionViewCell *)collectionView:(UICollectionView *)collectionView
   return cell;
 }
 
+- (UICollectionReusableView *)collectionView:(UICollectionView *)collectionView
+           viewForSupplementaryElementOfKind:(NSString *)kind
+                                 atIndexPath:(NSIndexPath *)indexPath
+{
+
+  // Must include below code snippet
+  id supplementaryView = [super collectionView:collectionView
+             viewForSupplementaryElementOfKind:kind
+                                   atIndexPath:indexPath];
+  if (supplementaryView) {
```

Comment:
This call to super seems strange to me.  
Why wouldn't we just return dequeueSupplementary... ?
Since this requires a nonnull return value, perhaps we fallback to super if our local implementation fails?